packages/components/src/components/FileUploader/utils/getStatus.test.ts

```diff
@@ -11,9 +11,19 @@ describe("getStatus", () => {
     expect(result).toMatch("success");
   });
 
-  it("returns error if not loading or success and with errorMessage", () => {
-    const result = getStatus({ progress: 100, errorMessage: "An error" });
-    expect(result).toMatch("error");
+  describe("returns error", () => {
```

Comment:
Both of these tests seem to do the same thing. We can probably revert this to what it was.